responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -17,10 +17,10 @@ dependencies:
     - pdfkit==1.0.0
     - plotly==5.6.0
     - kaleido==0.2.1
-    - azureml-core=={{latest-pypi-version}}
-    - azureml-dataset-runtime=={{latest-pypi-version}}
-    - azureml-mlflow=={{latest-pypi-version}}
-    - azureml-telemetry=={{latest-pypi-version}}
+    - azureml-core==1.48.0
+    - azureml-dataset-runtime==1.48.0
+    - azureml-mlflow==1.48.0
+    - azureml-telemetry==1.48.0
```

Comment:
Why were the `{{latest-pypi-version}}` tags removed?

Reply:
removed.